### PR TITLE
redid the functional components and redid the routes such that every …

### PR DESCRIPTION
**Pull request by Katiechi**
…route goes to its own page

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -9,23 +10,20 @@ import HowItWorks from './components/HowItWorks';
...
-        <About />
-        {/* Footer*/}
-        <Footer/>
-       </div> 
-       </div> );
+       
+      <Routes>
+          <Route path='/' element={<Masterhead/>} />
+          <Route path='/workings' element={<HowItWorks/>} />
+          <Route path='portfolio' element={<Portfolio/>} />
```
Comment (munenebrian) [inline client/src/App.js:20]: could you comment on what the routes have?

**Review by chibukasia — APPROVED** [review]
Good. I can see the changes I requested made

File: client/src/components/HowItWorks.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ import {FontAwesomeIcon} from "@fortawesome/react-fontawesome"
 import {faComputer} from "@fortawesome/free-solid-svg-icons"
 import Data from "../HowItWorks_data"
 //This is the how it works section and we have mapped over tghe single component
-const HowItWorks = () => {
```
Comment (munenebrian) [inline client/src/components/HowItWorks.js:6]: noted that you have changed all the conts, to function :+1:

**Review by munenebrian — APPROVED** [review]
The changes are :+1: for me, thou there is less of commenting to your code.

**Comment by chibukasia** [comment]
I'll merge the pull request but in the next branch you create, add some comments

**Pull request merged**